### PR TITLE
Added a guide for Windows

### PR DESCRIPTION
**Pull request by Ephraim-nonso**
**Status:  WORK IN PROGRESS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,17 +65,36 @@ units. One unit represents a payout of one token
...
-positions, positive numbers for FIXED.  There are no
+positions, positive numbers for FIXED. There are no
 zero positions.
 
-
 ## Getting started
 
+### On Windows
+
+Windows 10. Docker version 20.10.12, build e91ed57. Visual Studio Code v1.65.
```
Comment (dmarti) [inline README.md:75]: Can you add a link? Is this Docker Desktop for Windows?

File: README.md [hunk 2]
```diff
@@ -69,17 +65,36 @@ units. One unit represents a payout of one token
...
+
+Ensure to have downloaded Docker and follow the runs on their platform.
+
+On the installation of docker, run `docker build .`.
```
Comment (dmarti) [inline README.md:79]: Can you explain why this is needed as a separate step?  There is already a `docker build` command line in each of the 2 shell scripts below.

**Review by dmarti — CHANGES_REQUESTED** [review]
One general comment -- there are a lot of whitespace changes here that are not related to the text changes.

Can you make a commit that affects just the lines on which you are actually modifying the text?

File: README.md (hunk 1 above)
Comment (Ephraim-nonso) [reply README.md:75]: Alright, Don. I will effect the changes. I did observe a few things from my end, will push it soon.

**Comment by dmarti** [comment]
@Ephraim-nonso See suggestions on whitespace

https://app.codacy.com/gh/pinfactory/pinfactory/pullRequest

**Comment by dmarti** [comment]
Can you re-do the Windows script commit without the changes to the template?

Please keep commits doing one thing  and not changing multiple areas unless they depend on each other.

**Pull request closed without merging**